spec/controllers/dashboard/cards_controller_spec.rb

```diff
@@ -0,0 +1,79 @@
+require 'rails_helper'
+
+describe Dashboard::CardsController do
+
+  describe 'GET #index' do
+    it 'make array of all cards' do
+      user = create(:user_with_two_blocks_and_two_cards_in_each)
```

Comment:
why not in let(:user) block ?

Reply:
and why you didn't use treats, like a create(:user, :with_two_blocks_and_two_cards_in_each)